ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionEstimatedMemorySizeTest.java

```diff
@@ -64,7 +69,14 @@ public class PendingTransactionEstimatedMemorySizeTest extends BaseTransactionPo
   private static final Set<String> FRONTIER_ACCESS_LIST_CONSTANT_FIELD_PATHS =
       Sets.union(COMMON_CONSTANT_FIELD_PATHS, Set.of(".maxFeePerGas", ".maxPriorityFeePerGas"));
   private static final Set<String> VARIABLE_SIZE_PATHS =
-      Set.of(".chainId", ".to", ".payload", ".maybeAccessList");
+      Set.of(
```

Comment:
I suggest to add MAC for the supported OS above @EnabledOnOs({OS.LINUX, OS.MAC})

Reply:
done